core/src/consensus.rs

```diff
@@ -273,6 +293,14 @@ impl Tower {
         self.lockouts.root_slot
     }
 
+    pub fn last_lockout_vote_slot(&self) -> Option<Slot> {
```

Comment:
will this always just return the last item in self.lockouts.votes since the votes are always ordered smallest to largest?

Reply:
Yeah but the vote ordering doesn't matter since this implementation does a `max_by()`

Reply:
oh I meant it seems like the implementation can be simplified to `self.lockouts.votes.last().map(|v| v.slot)`

Reply:
Ah, right.  I was scared to make that sneaky assumption.   Feels like it could silently break on me in the future.  Should I just find some courage? 🦁

Reply:
hehe, true, but we could write a test that asserts this finds the largest!